executor/worker.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/vshn/k8up/observer"
 	"github.com/vshn/k8up/queue"
+
```

Comment:
File changed only by white spaces. Try not to do so. It keeps the risk of merge conflicts low. If you do want to fix formatting in general, do so in a PR on its own where only formatting fixes are done.

Comment by a burnt developer.

Reply:
I was changing other code in this file and therefore also performed minor reformatting. Later, I refactored the code and my changes were not relevant any longer and were removed. The formatting stayed. If they were to become a merge conflict, then I can still decide to replace them with the incoming change.